packages/utils/src/fhir/fhir-converter/scripts/run_integration_and_compare.sh

```
@@ -1,8 +1,11 @@
 #!/bin/bash
 # This script runs the integration-test.ts script without inserting to FHIR, and then runs the compare_total_resource_counts.sh script using the output from integration-test.ts.
 
+# Parse command line arguments and pass them to the integration test
+ARGS="$@"
```

Shellcheck (0.11.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)